### ISSUE TITLE
error during python setup.py install --boost_source=$HOME/museq/boost_1_57_0

### ISSUE THREAD

**Issue by msubirana** [shahcompbio/mutationseq#2]
In the finall step of the installation, during python setup.py install --boost_source=$HOME/museq/boost_1_57_0, I got this error:

Assembler messages:
Fatal error: can't create build/temp.linux-x86_64-2.7/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/libs/python/src/object_protocol.o: No such file or directory
error: command 'gcc' failed with exit status 1

Any advice?

**Comment by diljotgrewal** [comment]
Hey msubirana, 
Could you please post the full output from the setup.py command? 
Thanks

**Comment by diljotgrewal** [comment]
Hey, 
I just pushed a conda package for mutationseq as well. Could you please try:
`conda install -c shahcompbio museq`
you can check your museq install by running
`museq -h`
If it works then you can just skip all installation steps from the guide.
Cheers!

**Comment by msubirana** [comment]
Sorry for the delay, the output it's quite large but I attach below: 

```
(mutationseq) xxx@xxx:/imppc/labs/lplab/share/marc/repos/museq/mutationseq$ python setup.py install --boost_source=$ISHARE/marc/repos/museq/boost_1_57_0 

running install
running bdist_egg
running egg_info
writing pybam.egg-info/PKG-INFO
writing top-level names to pybam.egg-info/top_level.txt
writing dependency_links to pybam.egg-info/dependency_links.txt
reading manifest file 'pybam.egg-info/SOURCES.txt'
writing manifest file 'pybam.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-x86_64/egg
running install_lib
running build_ext
building 'pybam' extension
gcc -pthread -B /home/labs/lplab/msubirana/miniconda2/envs/mutationseq/compiler_compat -Wl,--sysroot=/ -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I./ -I./src -I/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0 -I./bamtools/src/ -I/home/labs/lplab/msubirana/miniconda2/envs/mutationseq/include/python2.7 -c src/pybam.cpp -o build/temp.linux-x86_64-2.7/src/pybam.o -Wno-unused-variable
cc1plus: warning: command line option '-Wstrict-prototypes' is valid for C/ObjC but not for C++
In file included from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/detail/wrap_python.hpp:50:0,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/detail/prefix.hpp:13,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/class.hpp:8,
                 from src/pybam.cpp:15:
/home/labs/lplab/msubirana/miniconda2/envs/mutationseq/include/python2.7/pyconfig.h:1220:0: warning: "_POSIX_C_SOURCE" redefined
 #define _POSIX_C_SOURCE 200112L
 ^
In file included from /usr/include/x86_64-linux-gnu/c++/4.9/bits/os_defines.h:39:0,
                 from /usr/include/x86_64-linux-gnu/c++/4.9/bits/c++config.h:430,
                 from /usr/include/c++/4.9/iostream:38,
                 from src/pybam.cpp:1:
/usr/include/features.h:230:0: note: this is the location of the previous definition
 # define _POSIX_C_SOURCE 200809L
 ^
In file included from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/detail/wrap_python.hpp:50:0,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/detail/prefix.hpp:13,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/class.hpp:8,
                 from src/pybam.cpp:15:
/home/labs/lplab/msubirana/miniconda2/envs/mutationseq/include/python2.7/pyconfig.h:1242:0: warning: "_XOPEN_SOURCE" redefined
 #define _XOPEN_SOURCE 600
 ^
In file included from /usr/include/x86_64-linux-gnu/c++/4.9/bits/os_defines.h:39:0,
                 from /usr/include/x86_64-linux-gnu/c++/4.9/bits/c++config.h:430,
                 from /usr/include/c++/4.9/iostream:38,
                 from src/pybam.cpp:1:
/usr/include/features.h:162:0: note: this is the location of the previous definition
 # define _XOPEN_SOURCE 700
 ^
In file included from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/refcount.hpp:9:0,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/to_python_indirect.hpp:19,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/converter/arg_to_python.hpp:10,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/call.hpp:15,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/object_core.hpp:14,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/object/class.hpp:9,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/class.hpp:13,
                 from src/pybam.cpp:15:
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/cast.hpp: In function 'void boost::python::detail::assert_castable(boost::type<T>*)':
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/cast.hpp:73:20: warning: typedef 'must_be_a_complete_type' locally defined but not used [-Wunused-local-typedefs]
       typedef char must_be_a_complete_type[sizeof(T)];
                    ^
In file included from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/detail/invoke.hpp:21:0,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/detail/caller.hpp:16,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/object/function_handle.hpp:8,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/converter/arg_to_python.hpp:19,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/call.hpp:15,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/object_core.hpp:14,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/object/class.hpp:9,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/class.hpp:13,
                 from src/pybam.cpp:15:
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/to_python_value.hpp: In member function 'PyObject* boost::python::detail::registry_to_python_value<T>::operator()(boost::python::detail::registry_to_python_value<T>::argument_type) const':
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/to_python_value.hpp:150:52: warning: typedef 'r' locally defined but not used [-Wunused-local-typedefs]
       typedef converter::registered<argument_type> r;
                                                    ^
In file included from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/data_members.hpp:15:0,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/class.hpp:17,
                 from src/pybam.cpp:15:
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/make_function.hpp: In function 'boost::python::api::object boost::python::detail::make_function_aux(F, const CallPolicies&, const Sig&, const keyword_range&, NumKeywords)':
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/make_function.hpp:58:32: warning: typedef 'assertion' locally defined but not used [-Wunused-local-typedefs]
           >::too_many_keywords assertion;
                                ^
In file included from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/class.hpp:20:0,
                 from src/pybam.cpp:15:
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/init.hpp: In constructor 'boost::python::init<T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13, T14>::init(const char*, const boost::python::detail::keywords<N>&)':
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/init.hpp:249:34: warning: typedef 'assertion' locally defined but not used [-Wunused-local-typedefs]
             >::too_many_keywords assertion;
                                  ^
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/init.hpp: In constructor 'boost::python::init<T0, T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13, T14>::init(const boost::python::detail::keywords<N>&, const char*)':
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/init.hpp:258:34: warning: typedef 'assertion' locally defined but not used [-Wunused-local-typedefs]
             >::too_many_keywords assertion;
                                  ^
In file included from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/class.hpp:24:0,
                 from src/pybam.cpp:15:
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/object/pickle_support.hpp: In static member function 'static void boost::python::detail::pickle_suite_registration::register_(Class_&, ...)':
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/object/pickle_support.hpp:110:31: warning: typedef 'error_type' locally defined but not used [-Wunused-local-typedefs]
           Class_>::error_type error_type;
                               ^
In file included from src/pybam.cpp:15:0:
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/class.hpp: In static member function 'static void boost::python::detail::error::virtual_function_default<T, Fn>::must_be_derived_class_member(const Default&)':
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/class.hpp:144:68: warning: typedef 'test1' locally defined but not used [-Wunused-local-typedefs]
             typedef typename assertion<is_polymorphic<T> >::failed test1;
                                                                    ^
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/class.hpp:146:81: warning: typedef 'test2' locally defined but not used [-Wunused-local-typedefs]
             typedef typename assertion<is_member_function_pointer<Fn> >::failed test2;
                                                                                 ^
In file included from src/pybam.cpp:17:0:
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/def.hpp: In function 'void boost::python::detail::def_from_helper(const char*, const F&, const Helper&)':
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/def.hpp:40:19: warning: typedef 'assertion' locally defined but not used [-Wunused-local-typedefs]
           >::type assertion;
                   ^
src/pybam.cpp: In member function 'void PyFasta::Open(const string&)':
src/pybam.cpp:495:29: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
   for (int refId = 0; refId < referenceNames.size(); refId++)
                             ^
src/pybam.cpp: In member function 'double PyFasta::GcContent(int, std::string&)':
src/pybam.cpp:643:21: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
   for (int i = 0; i < referenceSeq.length(); i++)
                     ^
src/pybam.cpp: In member function 'double PyFasta::Entropy(std::string&)':
src/pybam.cpp:670:21: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
   for (int i = 0; i < referenceSeq.length(); i++)
                     ^
gcc -pthread -B /home/labs/lplab/msubirana/miniconda2/envs/mutationseq/compiler_compat -Wl,--sysroot=/ -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I./ -I./src -I/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0 -I./bamtools/src/ -I/home/labs/lplab/msubirana/miniconda2/envs/mutationseq/include/python2.7 -c /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/libs/python/src/object_protocol.cpp -o build/temp.linux-x86_64-2.7/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/libs/python/src/object_protocol.o -Wno-unused-variable
cc1plus: warning: command line option '-Wstrict-prototypes' is valid for C/ObjC but not for C++
In file included from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/refcount.hpp:9:0,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/to_python_indirect.hpp:19,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/converter/arg_to_python.hpp:10,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/call.hpp:15,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/object_core.hpp:14,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/object_protocol.hpp:11,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/libs/python/src/object_protocol.cpp:6:
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/cast.hpp: In function 'void boost::python::detail::assert_castable(boost::type<T>*)':
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/cast.hpp:73:20: warning: typedef 'must_be_a_complete_type' locally defined but not used [-Wunused-local-typedefs]
       typedef char must_be_a_complete_type[sizeof(T)];
                    ^
In file included from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/detail/invoke.hpp:21:0,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/detail/caller.hpp:16,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/object/function_handle.hpp:8,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/converter/arg_to_python.hpp:19,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/call.hpp:15,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/object_core.hpp:14,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/object_protocol.hpp:11,
                 from /imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/libs/python/src/object_protocol.cpp:6:
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/to_python_value.hpp: In member function 'PyObject* boost::python::detail::registry_to_python_value<T>::operator()(boost::python::detail::registry_to_python_value<T>::argument_type) const':
/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/boost/python/to_python_value.hpp:150:52: warning: typedef 'r' locally defined but not used [-Wunused-local-typedefs]
       typedef converter::registered<argument_type> r;
                                                    ^
Assembler messages:
Fatal error: can't create build/temp.linux-x86_64-2.7/imppc/labs/lplab/msubirana/../share/marc/repos/museq/boost_1_57_0/libs/python/src/object_protocol.o: No such file or directory
error: command 'gcc' failed with exit status 1
```

**Comment by msubirana** [comment]
> Hey,
> I just pushed a conda package for mutationseq as well. Could you please try:
> `conda install -c shahcompbio museq`
> you can check your museq install by running
> `museq -h`
> If it works then you can just skip all installation steps from the guide.
> Cheers!

It is working perfectly! Thanks!
